### DISCUSSION TITLE
blog/2020-11-24-why-so-hard-tmux-copy-paste

### DISCUSSION THREAD

**Discussion by giscus** [crazychenz/vinnie.work#38] [vinnie.work Comments]
# blog/2020-11-24-why-so-hard-tmux-copy-paste

An adventure into how to enable the ability to copy paste from a
remote Linux machine to a local Windows machine from Tmux internal
buffers in a Windows Terminal.

https://www.vinnie.work/blog/2020-11-24-why-so-hard-tmux-copy-paste

**Comment by tsenapathy** [comment]
Thanks.  I didn't know that shift key would help :)
